hip-service/Startup.cs

```diff
@@ -40,6 +41,7 @@ public Startup(IConfiguration configuration)
                 .AddRouting(options => options.LowercaseUrls = true)
                 .AddControllers()
                 .AddNewtonsoftJson(options =>{});
+            services.AddSingleton<IConfiguration>(Configuration);
             services.AddControllers().AddJsonOptions(options =>
```

Comment:
can't we follow fluent compilation? services.AddNewtonsoftJson(options =>{})..AddControllers().AddJsonOptions